### DISCUSSION TITLE
现在无法报Bug

### DISCUSSION THREAD

**Discussion by Edric-Li** [element-plus/element-plus#9651] [General]
Feature type: **`Other`**

## Details

- Enhanced component name: `All`

## Description

无法选择 Element Plus的版本

### Additional comments

_(empty)_
![image](https://user-images.githubusercontent.com/50992790/188535987-59a03156-838b-487a-8023-0f6ccd0cc6d7.png)

**Comment by sxzz** [comment]
https://github.com/jsdelivr/jsdelivr/issues/18433

测试仅在香港节点有问题，尝试使用 CN（不开启 VPN）、JP、US 等节点

**Reply by sxzz** [reply]
Fixed https://github.com/jsdelivr/data.jsdelivr.com/issues/50

**Comment by Edric-Li** [comment]
好的，感谢
